examples/cypress/world.spec.js

```diff
@@ -0,0 +1,6 @@
+describe('world', () => {
```

Comment:
why is there a .ts and .js file both checked in? I suspect this one doesn't belong

Reply:
I did so to show how to load a `.js` file as a test and as a `.ts` file as a test respectively

Reply:
ah, okay then I would want them to have different filenames so it's clear one isn't derived from the other